### DISCUSSION TITLE
part of the standard nuclei-templates

### DISCUSSION THREAD

**Discussion by MRX7999** [projectdiscovery/nuclei#4492] [Q&A] [answered]
Hi! 
There is a problem when using part of the standard nuclei-templates. 
When using a folder with a different name and location than the standard one, part of the template cannot load. As far as I remember this was not the case in nuclei 2.* versions. Thank you in advance for your help. 
Command: ./nuclei -t ~/Desktop/Nuclei_3.1.0/templates/ -u testasp.vulnweb.com:443 -s medium,high,critical,unknown -etags dos,ssl,osint,dns,takeovers,token-spray,android,scada,apple,jobs -rl 10 -mr 10 -at batteringram -fr -no-stdin -nm -duc -je n_out.json > /dev/null && cat n_out.json
![Issue](https://github.com/projectdiscovery/nuclei/assets/42169049/795ff39e-78ed-45f7-b8ab-8f68aa8a5ea4)

**Comment by MetzinAround** [comment]
Thanks for the issue! I'm confused what you mean by "part of the template won't load." Do you mean it won't run every template or that some templates aren't run on the scan?

**Comment by MRX7999** [comment]
> Thanks for the issue! I'm confused what you mean by "part of the template won't load." Do you mean it won't run every template or that some templates aren't run on the scan? 

Hi, some templates aren't run on the scan.

**Comment by olearycrew** [comment, marked as answer]
Some templates won't run by default. That includes things like:

* **Headless templates**: These won't run without the `-headless` flag
* **Code templates**: These won't run without the `-code` flag
* **Fuzzing templates**: These require additional configuration and wont't run without additional configuration
* **DOS templates**: The Nuclei templates project houses a variety of templates which perform fuzzing and other actions which may result in a DoS against the target system (see [the list here](https://github.com/projectdiscovery/nuclei-templates/blob/master/.nuclei-ignore)). To ensure these templates are not accidentally run, they are tagged and excluded them from the default scan. These templates can be only executed when explicitly invoked using the -itags option.
